src/messages/mod.rs

```diff
@@ -136,6 +138,7 @@ impl Message {
         let signature = crypto::sign(&serialized, keypair);
         let src = SrcAuthority::Node {
             public_key: keypair.public,
+            age,
```

Comment:
same here take &Peer instead of key + age ?

Reply:
Peer contains `socket addr` as well, which no need to be in `SrcAuthority::Node` ?